### ISSUE TITLE
Request: Choose which mouse abilities are active

### ISSUE THREAD

**Issue by jamespharvey20** [vim/vim#4454]
`mouse` support allows:
* Left click in a text buffer to place cursor there, including selecting a buffer or tab, also works with operators.
* Drag status-line or vertical separator of a window to resize it.
* If enabled for visual mode, double, triple, and quadruple click selects word, line, and block.
* Scrolling.  (For me, works even with `mouse` off.  Maybe vim changed this, or maybe it's somehow [st](https://st.suckless.org/) or tmux.)

I think it would be a great feature to be able to choose which mouse abilities are active.

I'm sure some love moving the cursor with the mouse, but personally, I don't want it to do that.  i dislike moving the cursor with the mouse so much, that I keep `mouse` off entirely.

I wish I could enable only selecting with the mouse which buffer/tab is active, resizing, and scrolling.

I am using vim through a terminal.  I was first excited to see an option named `mousefocus`, but that doesn't do what I'm talking about, and only works in the GUI.  (I think that activates the window the mouse is under without even being clicked.)

It looks like someone else was wanting what I'm asking for here: https://superuser.com/questions/495652/vim-enable-mouse-scroll-only-not-cursor-positioning

Heptite answered that you can `<nop>` the left mouse button and double clicks.  But, the poster asked if there was a way to leave moving cursor position disabled, but still allow window/tab selection.  Heptite said he couldn't think of a way to do that.

**Comment by viplo** [comment]
Yes mouse capture s**ks, I had to disable mouse capture in a ~/.vimrc with set mouse= in it and if I need mouse in vim reactivate in file I need with :set mouse=a
